src/modules/scam/nft-scam.service.ts

```diff
@@ -246,12 +246,7 @@ export class NftScamService {
     const apiNfts = await this.mxApiService.getNftsByIdentifiers(nftsMissingFromDb?.map((x) => x.identifier));
     if (!apiNfts) return;
     let mappedNfts: Asset[] = [];
-    if (scamInfo.type === ScamInfoTypeEnum.none) {
-      mappedNfts = apiNfts?.map((x) => new Asset({ ...Asset.fromNft(x), scamInfo }));
-    } else {
-      mappedNfts = apiNfts?.map((x) => Asset.fromNft(x));
-      await this.pluginsService.computeScamInfo(mappedNfts);
-    }
+    mappedNfts = apiNfts?.map((x) => new Asset({ ...Asset.fromNft(x), scamInfo }));
```

Comment:
Declaration and initialization can be unified.

Reply:
done